weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -191,6 +191,9 @@ export const CallsTable: FC<{
 
   // Fetch the calls
   const calls = useCallsForQuery(entity, project, effectiveFilter);
+  if (calls.refetch) {
+    baseContext.refetchCalls = calls.refetch;
```

Comment:
This pattern would not be ideal in situations where there are 2 calls table on the screen at the same time (as there is when you open an eval). In general, it is not good practice to assign some state like this to a global symbol. Two comments:
1. If you wanted to do a pattern like this, you would want to have a react context provider somewhere above that allows children to "register" and "unregister" refetch triggers.
2. But even that is a little odd - and you might want to reach for a reducer pattern with listeners. 

Still, all these options are a bit odd - you really want someway to "broadcast" changes.

Got to run to dinner. Will respond later